.travis.yml

```diff
@@ -5,7 +5,7 @@ env:
   global:
     - ENV=conda
     - MINIMAL_ENV=false
-    - DEPS="numpy ipython scipy matplotlib traits natsort requests tqdm sympy dill h5py python-dateutil ipyparallel dask scikit-image pint numexpr statsmodels sparse imageio pyyaml PTable tifffile"
+    - DEPS="numpy ipython scipy matplotlib traits natsort requests tqdm sympy dill h5py python-dateutil ipyparallel dask scikit-image pint numexpr statsmodels sparse imageio pyyaml PTable tifffile==2020.2.16"
```

Comment:
Can we have >= for tiffile? I've had problems where some large facilities really don't want to install anything on their platforms as part of large python environments if it pins the version of anything that may cause incompatibility issues. If we don't pin it here then we'll know if we need to change it in future.

Reply:
This is pinned only for CI and I did this because I don't know a syntax which would work and be generic enough not to make the CI script more complicated for little reward. The idea is to make sure not to mixed pypi and conda package.

>  I've had problems where some large facilities really don't want to install anything on their platforms as part of large python environments if it pins the version of anything that may cause incompatibility issues.

It looks like defying the concept of python environment?! In any case, this PR doesn't change hyperspy installation requirement.

Reply:
I know it doesn't change the hyperspy installation requirement and that's part of my issue. If the CI is pinned like this but the install requirement is not - then we won't know that a future update of tiffile has broken the build (because it won't break the pinned CI) until a user reports it. So I think that the CI and install requirement should at least be the same and for me preferably be >=

Reply:
There is no simple syntax to do this with the current setup and as mentioned below, I would prefer to switch to use azure pipeline to simplify everything and increase CI capacity.

Reply:
Probably I am missing something, but, just in case, why not ">=2020.2.16" as suggested by @dnjohnstone ? That seems to be genuine ``conda`` syntax... Also I didn't get why we should not add the same constraint to ``setup.py``

Reply:
No I don't think that this syntax works with conda or we would need to change how CI is setup, which I don't want to do in this PR because I think we should most CI to azure pipeline. Currently, tifffile is installed from pypi because `defaults` is higher priority than `conda-forge` and `defaults` version is not recent enough, therefore it is updated by pip.
The idea here is to use this (temporary) workaround to avoid mixing wheels and conda packages.

Reply:
I've just tested it and it works with ``conda``, or at least it does the right thing and doesn't through any errors:

```bash
conda install -c conda-forge "tifffile>=2020.2.16"
```

Reply:
We've been using >= in orix for a while without any obvious issue.

https://github.com/pyxem/orix/blob/master/.travis.yml

Reply:
Great thanks, I may have tried with the first quotation marks in the wrong place! Giving a try now.

Reply:
Appveyor is not happy about it:
https://ci.appveyor.com/project/ericpre/hyperspy/builds/32194755/job/0a7ck2xagxtypcm7

```
if "%PLATFORM%"=="x64" set DEPS=%DEPS% "mrcz tifffile>=2019.7.26"
conda install -y %DEPS% %TEST_DEPS%
CondaValueError: invalid package specification: mrcz tifffile>=2019.7.26
Command exited with code 1

```


Do you have an example where it works on window?

Reply:
I think that the issue is that in Windows those characters must be escaped in Windows, maybe something like this would do: ``^>=2019.7.26``?

Reply:
I thought that the double quotation mark would take of it. Giving a try now!

Reply:
humm, no luck...

```
if "%PLATFORM%"=="x64" set DEPS=%DEPS% "mrcz tifffile^>=2019.7.26"
conda install -y %DEPS% %TEST_DEPS%
CondaValueError: invalid package specification: mrcz tifffile^^^^>=2019.7.26
Command exited with code 1
```

Reply:
I have not clue then, maybe a conda bug that affects only Windows. Anyway, to move on I suggest we pin for Windows and we keep ``>= `` in travis. Also adding some comments to explain why we've done all that will be helpful.

Reply:
Agree, this is done.